package.json

```
@@ -14,7 +14,7 @@
     "release:publish:dry-run": "lerna publish from-package --dist-tag latest --contents build --registry=\"http://localhost:4873/\"",
     "release:tag": "node scripts/releaseTag.mjs",
     "docs:api": "rimraf ./docs/pages/**/api-docs ./docs/pages/**/api && yarn docs:api:build",
-    "docs:api:build": "cross-env BABEL_ENV=development __NEXT_EXPORT_TRAILING_SLASH=true babel-node --extensions \".tsx,.ts,.js\" ./docs/scripts/buildApi.ts",
+    "docs:api:build": "ts-node ./docs/packages/api-docs-builder/buildApi.ts",
```

Perhaps it makes sense to move this script to the `api-docs-builder` package as a `build` script and instead replace this with
```suggestion
    "docs:api:build": "yarn workspace @mui-internal/api-docs-builder build",
```

---

It turns out that this creates a heap of problems as the scripts assume the `cwd` is at the repo root. It's fixable, of course, but let's do it in a separate PR.